src/utils/map-controller.js

```diff
@@ -204,14 +205,18 @@ export default class MapController {
   }
 
   toggleEvents(eventNames: Array<string>, enabled: boolean) {
+    // Bind the method here instead of in the constructor, so that inherited classes
+    // do not have to do binding themselves
```

Comment:
Not sure why you need to do this here. If you [bind in the constructor as was the case before](https://github.com/uber/react-map-gl/pull/651/files#diff-37b4265bef26974d812b2b2c97039331L59) then any class that extends this one should call `super()` it its constructor which should trigger the binding.

Reply:
Hmm I seem to have confused this with a legacy autobinder issue we had. Will revert to the good old way of binding.